snapcraft/internal/pluginhandler.py

```diff
@@ -685,6 +671,24 @@ def _migrate_files(snap_files, snap_dirs, srcdir, dstdir, missing_ok=False,
 
         fixup_func(dst)
 
+def _organize_filesets(fileset, base_dir):
+    for key in fileset:
+        src = os.path.join(base_dir, key)
+        dst = os.path.join(base_dir, fileset[key])
+
+        if os.path.isfile(dst):
+            raise EnvironmentError(
+                'Stepping over existing file for organization %r',
+                os.path.relpath(dst, base_dir))
```

Comment:
This message implies that this is just a warning and things will continue, but it really seems to be an error.